docs/content/guides/cell-features/clipboard.md

```diff
@@ -110,6 +109,107 @@ ReactDOM.render(<ExampleComponent />, document.getElementById('example1'));
 :::
 :::
 
+### Copy with headers
+
+Let the user copy the contents of column headers, by enabling the [additional options](@/api/options.md#copypaste-additional-options) of the [`CopyPaste`](@/api/copyPaste.md) plugin:
+
+| Context menu item       | Option                                                                    | Copied area                                                               |
+| ----------------------- | ------------------------------------------------------------------------- | ------------------------------------------------------------------------- |
+| Copy with headers       | [`copyColumnHeaders`](@/api/options.md#copypaste-additional-options)      | ![copy_with_headers]({{$basePath}}/img/copy_with_headers.png)             |
+| Copy with group headers | [`copyColumnGroupHeaders`](@/api/options.md#copypaste-additional-options) | ![copy_with_group_headers]({{$basePath}}/img/copy_with_group_headers.png) |
+| Copy headers only       | [`copyColumnHeadersOnly`](@/api/options.md#copypaste-additional-options)  | ![copy_headers_only]({{$basePath}}/img/copy_headers_only.png)             |
+
+To enable these features, configure the [`CopyPaste`](@/api/copyPaste.md) plugin like this:
+
+```js{2-4}
+copyPaste: {
+  copyColumnHeaders: true,
+  copyColumnGroupHeaders: true,
+  copyColumnHeadersOnly: true,
+}
+```
+
+Right-click on a cell to try it out:
+
+::: only-for javascript
+::: example #example2
+```js
+import Handsontable from 'handsontable';
+import 'handsontable/dist/handsontable.full.min.css';
+
+const container = document.querySelector('#example2');
+const hot = new Handsontable(container, {
+  data: [
+    ['A1', 'B1', 'C1', 'D1', 'E1', 'F1', 'G1', 'H1', 'I1', 'J1'],
+    ['A2', 'B2', 'C2', 'D2', 'E2', 'F2', 'G2', 'H2', 'I2', 'J2'],
+    ['A3', 'B3', 'C3', 'D3', 'E3', 'F3', 'G3', 'H3', 'I3', 'J3'],
+    ['A4', 'B4', 'C4', 'D4', 'E4', 'F4', 'G4', 'H4', 'I4', 'J4'],
+    ['A5', 'B5', 'C5', 'D5', 'E5', 'F5', 'G5', 'H5', 'I5', 'J5'],
+  ],
+  contextMenu: true,
+  copyPaste: {
+    copyColumnHeaders: true,
+    copyColumnGroupHeaders: true,
+    copyColumnHeadersOnly: true,
+  },
+  colHeaders: true,
+  rowHeaders: true,
+  height: 'auto',
+  nestedHeaders: [
+    ['H', { label: 'I', colspan: 2 }, { label: 'J', colspan: 2 }, { label: 'K', colspan: 2 }, { label: 'L', colspan: 2 }, 'M'],
+    ['N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W']
+  ],
```

Comment:
```suggestion
  nestedHeaders: [
    ['A', { label: 'B', colspan: 2 }, { label: 'C', colspan: 2 }, { label: 'D', colspan: 2 }, { label: 'E', colspan: 2 }, 'F'],
    ['G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P']
  ],
```
I'd follow the same rules as for the nested headers/collapsible columns examples where the header names start from A...Z